### PR TITLE
[Docs](bangc-ops): update pr_template

### PR DESCRIPTION
**Pull request by PetrelYy**
Thanks for your contribution and we appreciate it a lot. :rocket::rocket:

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

Are new test cases added? If so, please post the corresponding generator-PR link here.

## 3. Test Report

If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).

#### 3.1.1 Accuracy Acceptance Standard

For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Accuracy-Acceptance-Standard.md).

- static threshold
  - diff1
    - [ ] float32 mlu diff1 <= 1e-5
    - [ ] float32 mlu diff1 <= 3e-3
    - [ ] float16 mlu diff1 <= 3e-3
  - diff2
    - [ ] float32 mlu diff2 <= 1e-5
    - [ ] float32 mlu diff2 <= 3e-3
    - [ ] float16 mlu diff2 <= 3e-3
  - diff3
    - [ ] mlu diff3 == 0
    - [ ] mlu diff3_1 == 0
    - [ ] mlu diff3_2 == 0
- dynamic threshold
  - [ ] diff1: mlu diff1 <= max(baseline diff1 * 10, static threshold)
  - [ ] diff2: mlu diff2 <= max(baseline diff2 * 10, static threshold)
  - [ ] diff3: mlu diff3 <= max(baseline diff3 * 10, static threshold)
    - float32, threshold = 1e-5
    - float16, threshold = 1e-3

#### 3.1.2 Operator Scheme checklist

- Supported hardware
  - [ ] MLU370
  - [ ] MLU590
- Job types
  - [ ] BLOCK
  - [ ] UNION1
  - [ ] UNION2
  - [ ] UNION4
  - [ ] The operator will dynamically select the most suitable task type, for example, UNION8

#### 3.2.1 Accuracy Test

If you have checked the following items, please tick the relevant box.

- [ ] Data type test (e.g. float32/int8)
- [ ] Multi-dimensional tensor test
- [ ] Layout test
- [ ] Different size/integer remainder end segment/alignment misalignment test
- [ ] Zero dimensional tensor test/zero element test
- [ ] stability test
- [ ] Multiple platform test
- [ ] Gen_case module test, see: [Gencase-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/Gencase-User-Guide-zh.md)
- [ ] Nan/INF tests 
- [ ] Bug fix tests
- [ ] For memory leak check details, see: [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md)
- [ ] For code coverage check details, see: [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md)
- [ ] For I/O calculation efficiency check details, see: [MLU-OPS-Performance-Acceptance-Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md)

#### 3.2.2 Parameter Check

Test Point-1: `When a new operator is submitted, the test points are given and the test results are stated`. Acceptance Standard: `Normal error`.
```bash
Please fill your test results(Error Message) in here, ...
```

Test Point-2: `Whether illegal parameters are passed`. Acceptance Standard: `Normal error`.
```bash
Test results...
```

### 3.3 Performance Test

See [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md) for details.

Platform：MLU370

```bash
# The test results should contain Op name, Shape, Data type,  
#   MLU Hardware Time(us), MLU Interface Time(us), MLU IO Efficiency, 
#   MLU Compute Efficiency, and Mlu Workspace Size(Bytes)
# 
# for example:
#
# ----------- case0 -----------
# case0
# [Op name                ]: abs
# [Shape                  ]: input.shape=[1024,1024,3,4], output.shape=[1024,1024,3,4]
# [Data type]             ]: float32
# [MLU Hardware Time      ]: 15728 (us)
# [MLU Interface Time     ]: 369.008 (us)
# [MLU IO Efficiency      ]: 0.23275
# [MLU Compute Efficiency ]: 0.5
# [Mlu Workspace Size     ]: -1 (Bytes)
# 
# ----------- case1 -----------
# ...
```

Platform：MLU590
```bash
# ----------- case0 -----------
# ----------- case1 -----------
# ...
```

### 3.4 Summary Analysis

Please give a brief overview here, if you want to note and summarize the content.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -18,84 +20,103 @@ If you want to know how to do operator testing, you can see [GTest-User-Guide-zh
 
 For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Accuracy-Acceptance-Standard.md).
 
-- [ ] diff1: diff1 <= 3e-3
-- [ ] diff2: diff2 <= 3e-3
+- static threshold
+  - diff1
+    - [ ] flaot32 mlu diff1 <= 1e-5
```
Comment (Unireverse) [inline .github/pull_request_template.md:25]: float写错了

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -18,84 +20,103 @@ If you want to know how to do operator testing, you can see [GTest-User-Guide-zh
...
-|        6       |Data type(half/float)                 |           half / float etc           |
-|        7       |Whether there is size limit           |                                      |
+- Supported hardware
+  - [ ] MLU370
+  - [ ] MLU590
+- Job types
+  - [ ] BLOCK
+  - [ ] UNION1
+  - [ ] UNION4
+  - [ ] UNION_MAX
```
Comment (mahxn0) [inline .github/pull_request_template.md:51]: 这个UNION_MAX类型有解释吗

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -18,84 +20,103 @@ If you want to know how to do operator testing, you can see [GTest-User-Guide-zh
...
+
+### 3.2 Accuracy Test
 
-#### 3.1.3 New Feature Test
+#### 3.2.1 Accuracy Test
 
 If you have checked the following items, please tick the relevant box.
 
-- [ ] Data type test
+- [ ] Data type test (half/float/int8)
```
Comment (mahxn0) [inline .github/pull_request_template.md:59]: 这里感觉不用特殊注明类型，要写的话还有int32 ?

Comment (Unireverse) [inline .github/pull_request_template.md:59]: (e.g. half/float/int8)

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -8,6 +8,8 @@ Please describe your motivation and the goal you want to achieve through this pu
 
 Please briefly describe what modification is made in this pull request, and indicate where to make the modification.
 
+Are new test cases added? If so, please post the corresponding generator-PR link here.
+
 ## 3. Test Report
```
Comment (DanieeelLiu) [inline .github/pull_request_template.md:13]: 是不是要给个例子？

File: .github/pull_request_template.md (hunk 2 above)
Comment (PetrelYy) [reply .github/pull_request_template.md:51]: 没有解释，或者有啥建议？

File: .github/pull_request_template.md (hunk 3 above)
Comment (PetrelYy) [reply .github/pull_request_template.md:59]: 这里half删掉了，和上文不一致。 改为了  (e.g. float32/int8)

Comment (PetrelYy) [reply .github/pull_request_template.md:59]: 特殊类型是指？

Comment (PetrelYy) [reply .github/pull_request_template.md]: 已修改

File: .github/pull_request_template.md (hunk 2 above)
Comment (mahxn0) [reply .github/pull_request_template.md:51]: 感觉union_x合适，根据规模选择u1/u2/u4/u8/u16

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -18,84 +20,103 @@ If you want to know how to do operator testing, you can see [GTest-User-Guide-zh
...
+    - [ ] float32 mlu diff1 <= 3e-3
+    - [ ] float16 mlu diff1 <= 3e-3
+  - diff2
+    - [ ] float32 mlu diff2 <= 1e-5
+    - [ ] float32 mlu diff2 <= 3e-3
+    - [ ] float16 mlu diff2 <= 3e-3
+  - diff3
+    - [ ] float32 mlu diff3 == 0
+    - [ ] float16 mlu diff3 == 0
+- dynamic threshold
```
Comment (DanieeelLiu) [inline .github/pull_request_template.md:35]: 精度文档没提动态阈值，这里是不是就不要了

**Review by Unireverse — APPROVED** [review]

**Review by carcel-yu — APPROVED** [review]

**Pull request merged**